Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -207,10 +209,10 @@ protected override Size ArrangeOverride(Size finalSize)
                 var paddingEnd = new UvMeasure(Orientation, Padding.Right, Padding.Bottom);
                 var position = new UvMeasure(Orientation, Padding.Left, Padding.Top);
 
-                double currentV = 0;
+                var currentRow = new Row();
```

Comment:
Can we eliminate the need to measure twice by storing the rows collection as a member variable and use it during the arrange. Maybe even reuse the Rows objects every measure instead of creating new objects

Reply:
In some (rare) cases, we `Arrange()` may not get exactly the size that was returned by `Measure()`. It should be safer to do the measure twice even if most of the time, it is just doing exactly the same work. That being said, we can try to have something "smart" that will check if the measure's result can be reused or not. I'll try to implement it later.

Reply:
I've updated the code so now, `Arrange()` will reuse the values computed during the measure pass.